openage/launcher/mod.py

```diff
@@ -0,0 +1,64 @@
+import toml
+
+
+class ModInfo:
+    """
+    Parses and exposes info about a mod
+    """
+    def __init__(self, file_path=None, url=None):
+        if file_path is not None:
+            with open(file_path, 'r') as f:
+                info = toml.load(f)
+        elif url is not None:
+            info_str = some_black_magic(url)    # TODO: pseudocode
+            info = toml.loads(info_str)
+        else:
+            raise Exception
+ 
+        self.info = info
+
+        # for convenience
+        self.uid = self.info['uid']
+        self.version = self.info['version']
+        self.author = self.info['author']
+        self.confl = self.info['conflicts']
+        self.req = self.info['requires']
```

Comment:
Writing variable names out will improve readability a lot and is not even that complicated if you use an IDE :D

`confl` -> `conflicts`
`req` -> `required`

Reply:
This was to avoid name clashes with the methods "conflicts" and "requires". Which will not be a problem anymore when I take into account your suggestion below here :)